modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -350,8 +393,10 @@ protected function render() {
 			$tab_title = $a11y_improvements_experiment ? $item['tab_title'] : '<a href="">' . $item['tab_title'] . '</a>';
 			$tab_title_mobile_setting_key = $this->get_repeater_setting_key( 'tab_title_mobile', 'tabs', $tab_count );
 
+			$tab_id = '' === $item['element_id'] ? 'elementor-tab-title-' . $id_int . $tab_count : $item['element_id'];
```

Comment:
It should handle also if it's not set.
```suggestion
			$tab_id = empty( $item['element_id'] ) ? 'elementor-tab-title-' . $id_int . $tab_count : $item['element_id'];
```